Makefile

```diff
@@ -46,6 +46,22 @@ format:
 	./mvnw compile test-compile scalafix:scalafix -Psemanticdb $(PROFILES)
 	./mvnw spotless:apply $(PROFILES)
 
+# build native libs for amd64 architecture Linux/MacOS on a Linux/amd64 machine/container
+core-amd64-libs:
+	cd native && cargo build -j 2 --release
```

Comment:
Don't we need to specify `target` for this?

Reply:
This will build the binary for the same architecture as the machine. So no need to specify target.